Dockerfile

```
@@ -6,10 +6,14 @@ RUN apt update && apt install -y \
     g++-aarch64-linux-gnu \
     g++-arm-linux-gnueabi \
     g++-powerpc-linux-gnu \
+    g++-mips-linux-gnu \
+    g++-mipsel-linux-gnu \
```

These packages aren't sorted. I don't know if it's that big a deal, but it might be preferable.
```suggestion
    g++-aarch64-linux-gnu \
    g++-arm-linux-gnueabi \
    g++-mips-linux-gnu \
    g++-mipsel-linux-gnu \
    g++-powerpc-linux-gnu \
```